nsxt/tests_utils.go

```diff
@@ -80,3 +83,185 @@ func copyStatePtr(t **terraform.State) resource.TestCheckFunc {
 		return nil
 	}
 }
+
+// Create and delete CA and client cert for various tests
+func testAccNSXCreateCert(t *testing.T, name string, certPem string, certPK string, certType string) string {
+	nsxClient, err := testAccGetClient()
```

Comment:
This is failing because testAccGetClient was not found. Is that is another patch? maybe need to rebase

Reply:
Weird.. fixed